patches/components-crash-content-app-crashpad_mac.mm.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/crash/content/app/crashpad_mac.mm b/components/crash/content/app/crashpad_mac.mm
+index eb82f44053cec67f7e98c798d8e1e58228aa7fe6..3e6189b8dd2106d79059268b1fc2d445ff93da92 100644
+--- a/components/crash/content/app/crashpad_mac.mm
++++ b/components/crash/content/app/crashpad_mac.mm
+@@ -55,6 +55,8 @@ base::FilePath PlatformCrashpadInitialization(bool initial_client,
+       // Only allow the possibility of report upload in official builds. This
+       // crash server won't have symbols for any other build types.
+       std::string url = "https://clients2.google.com/cr/report";
```

Comment:
I think we should handle all of these by redirecting in the network delegate

Reply:
Network delegate not used by crashpad.